Formula/borg.rb

```diff
@@ -0,0 +1,31 @@
+# Documentation: https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class Borg < Formula
+  desc "A terminal based search engine for bash commands"
```

Comment:
Please remove the indefinite article: `"Terminal-based search engine for Bash commands"`.